cfme/tests/services/test_catalog_item.py

```diff
@@ -151,7 +151,6 @@ def test_edit_tags_catalog_item(catalog_item):
     catalog_item.remove_tag(tag)
 
 
-@pytest.mark.meta(blockers=[BZ(1531512, forced_streams=["5.9", "5.10", "upstream"])])
```

Comment:
This bug is still open, @niyazRedhat can you take a look?

Reply:
There is a comment that its present in 5.10 too, but the BZ has not been cloned to 5.10, so the BZ doesn't block. We'll need to use forced_streams here to block, unless/until the BZ is updated.  @niyazRedhat please add a NEEDINFO to that BZ to get it updated with 5.10 clone.

Reply:
To be clear - I'm adding forced_streams back in this PR.